spec/lib/logstasher/railtie_spec.rb

```diff
@@ -18,6 +18,22 @@ class ::LogStasher::RailtieApp < ::Rails::Application
 describe ::LogStasher::Railtie do
   let(:config) { described_class.config.logstasher }
 
+  describe 'logstasher should touch log file to prevent creation message by default' do
+    it 'should configure LogStasher' do
+      expect(::FileUtils).to receive(:touch)
+      ActiveSupport.run_load_hooks(:before_initialize)
+    end
+  end
+
+  describe 'logstasher should NOT touch log file if silence disabled' do
+    it 'should configure LogStasher' do
+      config.silence_creation_message = false
```

Comment:
You may want to mock this with a test or use a before/ after clause because setting this global to false could have some issues with other tests who are expecting it to be true.

```ruby
expect(config).to receive(:silence_creation_message).and_return(false)
```

Or for the before/after approach:

```ruby
before { config.silence_creation_message = false }
after { config.silence_creation_message = true }

it 'should configure LogStasher' do
  # ...
end
```

Reply:
updated to before/after approach